python/lsst/ts/externalscripts/maintel/take_twilight_flats_comcam.py

```diff
@@ -0,0 +1,209 @@
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_externalscripts/take_twilight_flats_comcam.yaml
+            title: TakeTwilightFlatsComCam v1
+            description: Configuration for TakeTwilightFlatsComCam.
+            type: object
+            properties:
+              filter:
+                description: Filter name or ID.
+                anyOf:
+                  - type: string
+                  - type: integer
+                    minimum: 1
+                  - type: "null"
+            required: ["filter"]
+            additionalProperties: false
+        """
+        schema_dict = yaml.safe_load(schema_yaml)
+
+        base_schema_dict = super(TakeTwilightFlatsComCam, cls).get_schema()
+
+        for prop in base_schema_dict["properties"]:
+            schema_dict["properties"][prop] = base_schema_dict["properties"][prop]
+
+        return schema_dict
+
+    async def get_sky_counts(self) -> float:
+        """Abstract method to get the median sky counts from the last image.
+
+        Returns
+        -------
+        float
+            Sky counts in electrons.
+        """
+        timeout = 30
+        query = f"SELECT * from cbd_lsstcomcam.visit1_quicklook where exposure_id = {self.latest_exposure_id}"
+        item = "post_isr_pixel_median_median"
+        sky_counts = self.client.wait_for_item_in_row(query, item, timeout)
+        return sky_counts
+
+    def get_instrument_name(self) -> str:
+        """Get instrument name.
+
+        Returns
+        -------
+        instrument_name: `string`
+        """
+        return "LSSTComCam"
+
+    def get_instrument_configuration(self) -> dict:
+        return dict(
+            filter=self.config.filter,
+        )
+
+    def get_instrument_filter(self) -> str:
+        """Get instrument filter configuration.
+
+        Returns
+        -------
+        instrument_filter: `string`
+        """
+        return f"{self.config.filter}"
+
+    async def track_radec_and_setup_instrument(self, ra, dec):
+        """Method to set the instrument filter and slew to desired field.
+
+        Parameters
+        ----------
+        ra : float
+            RA of target field.
+        dec : float
+            Dec of target field.
+        """
+        current_filter = await self.comcam.get_current_filter()
+
+        self.tracking_started = True
+
+        if current_filter != self.config.filter:
+            self.log.debug(
+                f"Filter change required: {current_filter} -> {self.config.filter}"
+            )
+            await self._handle_slew_and_change_filter()
+        else:
+            self.log.debug(
+                f"Already in the desired filter ({current_filter}), slewing and tracking."
+            )
+
+        await self.mtcs.slew_icrs(
+            ra=ra,
+            dec=dec,
```

Comment:
I don't think you care about the rotation angle, so long as it is tracking right? In this case, use `rot_type=RotType.PhysicalSky`.